data/coalition/coalition outfits.txt

```diff
@@ -239,15 +239,15 @@ outfit "Refueling Module"
 
 outfit "Shield Refactor Module"
 	category "Systems"
-	cost 2843000
+	cost 8324000
 	thumbnail "outfit/shield refactor module"
-	"mass" 7
-	"outfit space" -7
-	"hull repair rate" -.2
+	"mass" 11
+	"outfit space" -11
+	"hull repair rate" -.25
```

Comment:
This change specifically I would keep, since it fixes the specific slightly over-strong combination of this outfit and korath syscores that the previous value of -.2 (-12) had.